master/internal/core.go

```diff
@@ -1173,7 +1173,10 @@ func (m *Master) Run(ctx context.Context) error {
 	// Catch-all for requests not matched by any above handler
 	// echo does not set the response error on the context if no handler is matched
 	m.echo.Any("/*", func(c echo.Context) error {
-		return echo.ErrNotFound
+		id := fmt.Sprintf("%s %s", c.Request().Method, c.Request().URL.Path)
```

Comment:
is it too much to log these? maybe at debug level?

Reply:
yeah I would log these at debug level - I think the error message is clear enough to the user that the endpoint doesn't exist